app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -8,6 +8,12 @@ import { Label, Menu, Icon, Popup } from "semantic-ui-react";
 import numberWithCommas from "../helpers/strings";
 import { getTaxonName } from "../helpers/taxon";
 import ThresholdMap from "./utils/ThresholdMap";
+import {
+  computeThresholdedTaxons,
+  isTaxonIncluded,
+  getTaxonSortComparator,
+  getCategoryAdjective
+} from "./utils/report";
```

Comment:
If theses are report specific, I believe it makes more sense to put this inside the report view, instead of utils.

Reply:
We also should move `PipelineSampleReport` to `views/report` and maybe rename it in one of your next PRs.

Reply:
Makes sense. I will move the utils file to inside report view.